### ISSUE TITLE
Support for Scala 2.13

### ISSUE THREAD

**Issue by psisoyev** [doriordan/skuber#283]
Is there a plan to add support of Scala 2.13? 

Some libraries need to be updated though: 
```
[warn] 	:: com.typesafe.akka#akka-http_2.13;10.1.3: not found
[warn] 	:: com.typesafe.akka#akka-stream_2.13;2.5.14: not found
[warn] 	:: com.typesafe.play#play-json_2.13;2.7.2: not found
[warn] 	:: org.specs2#specs2-core_2.13;4.3.2: not found
[warn] 	:: com.typesafe.akka#akka-stream-testkit_2.13;2.5.14: not found
[warn] 	:: org.scalatest#scalatest_2.13;3.0.5: not found
```

As I can see these libraries already support Scala 2.13 in their newer versions

**Comment by psisoyev** [comment]
Decided not to wait for your response:
#284

**Comment by doriordan** [comment]
Implemented in #287
